src/WCSimConstructCylinder.cc

```diff
@@ -90,6 +90,18 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructCylinder()
   if (WCAddGd)
   {water = "Doped Water";}
 
+  // Optionally switch on the checkOverlaps. Once the geometry is debugged, switch off for 
+  // faster running. Checking ALL overlaps is crucial. No overlaps are allowed, otherwise
+  // G4Navigator will be confused and result in wrong photon tracking and resulting yields.
+  // ToDo: get these options from .mac
+  checkOverlaps = true;   
+  checkOverlapsPMT = false; 
```

Comment:
Should the checkOverlapsPMT be true by default? The places where it shows up are true in the current code.